lib/std/fmt/parse_float/FloatInfo.zig

```diff
@@ -106,6 +106,26 @@ pub fn from(comptime T: type) Self {
             .min_exponent_round_to_even = -4,
             .max_exponent_round_to_even = 23,
         },
+        f80 => .{
+            // Fast-Path
+            .min_exponent_fast_path = -27,
+            .max_exponent_fast_path = 27,
+            .max_exponent_fast_path_disguised = 46,
+            .max_mantissa_fast_path = 2 << std.math.floatMantissaBits(T),
+            // Slow + Eisel-Lemire
+            .mantissa_explicit_bits = std.math.floatFractionalBits(T),
```

Comment:
I think it is clearer if we change the other float types to use `floatFractionalBits` instead of `floatMantissaBits` as done here. The value is the same for the other IEEE types.